lib/src/ui/puzzle/puzzle_dashboard_screen.dart

```diff
@@ -81,6 +82,7 @@ class _PuzzleDashboardScreenState extends ConsumerState<PuzzleDashboardScreen> {
   }
 
   Future<void> _refreshData() {
+    ref.invalidate(puzzleHistoryProvider);
```

Comment:
See: https://github.com/lichess-org/mobile/blob/main/lib/src/ui/account/profile_screen.dart#L138

on how to refresh several providers at once.